includes/object-cache.php

```diff
@@ -151,6 +151,20 @@ function wp_cache_flush( $delay = 0 ) {
     return $wp_object_cache->flush( $delay );
 }
 
+/**
+ * Invalidate all items in the cache with the `WP_REDIS_PREFIX` prefix.
+ *
+ * @param int $delay  Number of seconds to wait before invalidating the items.
+ *
+ * @return bool       Returns TRUE on success or FALSE on failure.
+ */
+function wp_cache_flush_group($delay = 0)
```

Comment:
This function accepts the group name:

https://github.com/WordPress/wordpress-develop/blob/832b6c35f36d5f1ca84feadc49b0e5424af0ad6c/src/wp-includes/cache-compat.php#L155-L185